examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal

```diff
@@ -39,14 +39,14 @@ service<grpc:Service> Chat bind ep {
         }
     }
 
-    //This resource is triggered when some error has been happen at server end
+    //This resource is triggered when an error has occurred at the server end.
     onError(endpoint caller, error err) {
         if (err != ()) {
             io:println("Something unexpected happens at server : " + err.message);
         }
     }
 
-    //This resource is triggered when client notify service that it has finish send requests
+    //This resource is triggered when the client notifies the service that it has finished sending requests.
```

Comment:
shall we make it like "when server receives complete notification from the caller. Caller sends complete notification when it has finished sending messages"